packages/now-cli/src/util/errors-ts.ts

```diff
@@ -1236,3 +1237,16 @@ export class BuildError extends NowError<'BUILD_ERROR', {}> {
     });
   }
 }
+
+export class NpmInstallError extends NowBuildError {
```

Comment:
I'm curious if we still need this pattern at all. Why a new class for each error? Are you planning to catch it somewhere? Since the `message` is passed in, it doesn't seem very reusable.

Reply:
a95312d